definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -5,5 +5,12 @@
     "CaseEventID": "tempEditDerivedAddresses",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
```

Comment:
Could you please add it to the version with the feature-toggle name?

Reply:
the thing is that it's not helping. When nonprod files are just feature toggled when release - just copy over to the "core" file.

If we had there hacks and workarounds it will require additional investigation and the whole process will be more error-prone.

Reply:
Then this file should be just nonprod ;)
definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

Reply:
got you